dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -21,6 +25,14 @@
               :model-value="companyID"
               disabled="true"
             />
+            <FormKit
+              type="hidden"
```

Comment:
The reporting period of an uploaded data set is not shown on the display page.

Reply:
I don't fully understand this comment.

The reporting period is displayed on the display page in the column headers of each lksg dataset.

And here in the form, the field is hidden, because it is auto-filled with the year of the dataDate.

Reply:
Siegfried and I had a phone call where he explained this.
This is broken and needs a fix.

Reply:
yes it works now